root/user/profile.tt

```diff
@@ -91,6 +91,12 @@
             [% END %]
         [% END %]
 
+        [% IF c.user_exists AND (viewing_own_profile OR c.user.is_account_admin);
+             WRAPPER property name=l('Last password login:');
```

Comment:
Yeah, at least in the code we call the other way to get a Catalyst session a “cookie login”. Normal users will understand it like Gentlecat did, so we could probably say just “Last login” and hope that account admins will still remember this can be quite different from “Last activity”.